### PR TITLE
Events nav dropdown now holds upcoming events

### PR DESCRIPTION
**Pull request by courte**
An easy way to find upcoming events -- the nav bar!

Not sure about adding this database-dependent `before_filter` to every controller - if it creates unnecessary noise - but it works!

*Nav bar with upcoming events*
screenshot 2015-02-12 16 31 58

### CODE REVIEW

File: app/views/layouts/_navigation.html.erb [hunk 1]
```diff
@@ -16,6 +16,11 @@
         <li><%= link_to "Jobs", jobs_path %></li>
         <li class="has-dropdown"><%= link_to "Events", events_path %>
           <ul class="dropdown">
+            <% if @nav_events %>
+              <% @nav_events.each do |nav_event| %>
+                <li><%= link_to nav_event.name, event_path(nav_event) %></li>
+              <% end %>
+            <% end %>
             <li><%= link_to "Women Coders Dinner", 'https://www.codemontage.com/events/amplify-women-coders-dinner' %></li>
```
Comment (DBNess) [inline app/views/layouts/_navigation.html.erb]: let's remove this women coders dinner link -- this was a weird hardcoded link to the first dinner anyway.

File: app/views/layouts/_navigation.html.erb [hunk 2]
```diff
@@ -16,6 +16,11 @@
...
             <li><%= link_to "Coder Day of Service", coder_day_path %></li>
```
Comment (DBNess) [inline app/views/layouts/_navigation.html.erb]: We can remove "Coder Day of Service" link, too, and maybe replace with "Past Events" link to the /events index page.

**Comment by courte** [comment]
*Past Events instead of hardcoded ones*
screenshot 2015-02-13 10 20 25

**Comment by courte** [comment]
*Updated nav bar!*
screenshot 2015-02-13 10 41 03

**Comment by DBNess** [comment]
:+1::+1::+1::sunglasses:

**Pull request merged**